Marlin/src/gcode/calibrate/G33.cpp

```diff
@@ -338,7 +341,7 @@ static void calc_kinematics_diff_probe_points(float z_pt[NPP + 1], abc_float_t d
 }
 
 static float auto_tune_h() {
-  const float r_quot = delta_calibration_radius / delta_radius;
+  const float r_quot = delta_calibration_radius() / delta_radius;
```

Comment:
r_quot can be delta_calibration_radius / delta_radius or 0.9*delta_calibration_radius / delta_radius depending on probe grid